### ISSUE TITLE
Resolve comments from Gonzalez lab meeting

### ISSUE THREAD

**Issue by gottacatchenall** [PoisotLab/ms_network_prediction#89]
- [x] example section merged
- [x] open data table
- [x] conclusion
- [x] Introductory paragraph better summarizes scope of paper
- [x] ~Cut everything is connected figure~ completed in #90
- [x] ~Triangle scale figure (?)~ not happening
- [x] Model Validation section include accuracy measurement from example
- [x] Figure for example created #90 #91

**Comment by TanyaS08** [comment]
Some additional notes:

I think the triangle scale figure thing was more of a potential way to rework the everything is connected figure - I think given the current approach with the example I don't see a place to shoehorn that in?

**Concluding section/notes** I made a note of expanding on the working example and how we can build/add to it. I think framing it as a looking forward type of thing could work

**Open Data** Do we bring up the 'issue' of structured/unstructured data? If yes I think its 1/2 points under the what is enabling/open tools and methods i.e. section 3.2 would be a good place to bring it up - we can point to where this has been overcome/circumvented

Remote sensing under open tools and methods (??)

**Comment by gottacatchenall** [comment]
okay i've added those to the list as well, if you think the restructuring in #90 is good i will edit the text to reflect that and then merge, and then we can get each item done as a single pr

**Comment by TanyaS08** [comment]
Yup works for me feel free to re-restructure and merge - we can just tag the relevant PRs in the issue here like we did previously yeah?

**Comment by gottacatchenall** [comment]
sounds good, about to merge the reorganize, and i'll probably add some of tim's plots to the second panel of the case study fig later

**Comment by tpoisot** [comment]
I will work on the tools/data section a bit, I don't like tables

**Comment by tpoisot** [comment]
Let me rephrase: I _have worked_ on this section a bit and I don't think we should cut ir

**Comment by gottacatchenall** [comment]
okay thats cool, so what is left is a conclusion and a couple of sentences in the intro paragraph to describe that we give an example and look at the path forward?

**Comment by tpoisot** [comment]
I got the conclusion - just an update, we are now at a solid 8.9k words

**Comment by tpoisot** [comment]
13k if we count references

**Comment by gottacatchenall** [comment]
sounds good--- i can do an editing pass aimed at condensing language later and then @TanyaS08 can take a look, and hopefully we can have it sent out by tomorrow

**Comment by tpoisot** [comment]
conclusion done - I'll let the two of you work on the intro, and then let's aim to send it tomorrow

**Comment by TanyaS08** [comment]
I re-read the thread in (what I think was the now closed PR #90) and the only thing I'm a bit confused about is the state of what is now figure 1 - which from what I can gather is a question of do we have the schematic with the data? I think maybe just sticking to the 'data' figure makes sense as its less daunting than having both and the schematics are 'meaningless' until you come across figure 2 and even then I don't know if it makes it simpler? I think leaving it open to the floor is probably the way to go but that's my 2c

**Comment by TanyaS08** [comment]
Oh and given the now inclusion of a data-driven example are we still married to the idea of going to Oikos or are we re-opening the discussion to maybe look at e.g. Ecology Letters? (at ~9K of content we should just go for eco monographs 😉 )

**Comment by gottacatchenall** [comment]
my thinking with including the current figure 1 is that to a reader that is not at all familiar with networks, i can imagine them being confused (asking "where is the network?", not understanding what prediction is adding)  when looking the example if they are not familiar with embedding as a method of visualization (we don't address embeddings until the _Primer_ section).

**Comment by gottacatchenall** [comment]
i think we can feasibly get the main text to ~8k and have a good shot a oikos.  ecography has a _review and synthesis_ type (max 7500) which i also think could work.

**Comment by gottacatchenall** [comment]
i think getting it below 6k for letters would be tough

**Comment by TanyaS08** [comment]
Figures: True (regarding unfamiliarity) I think then merging the two figures as you had it in the original conversation works better then, to capture the mirroring? AS opposed to the two objects it is currently

Wordcount: Also true

**Comment by gottacatchenall** [comment]
from ecology letters (both 7500 word limit)
> **Ideas and Perspectives**. Ecology Letters is particularly interested in novel essays expressing new ideas and perspectives that will appeal to a wide ecological audience. It is important that Ideas and Perspectives be focused on a topic of current interest. We are interested in new ideas, emerging frameworks, and controversial perspectives on hot areas of research. There is a need to present a view that is sufficiently complete to convince reviewers of the value of the contribution. There is the same expectation for the novelty of Ideas and Perspectives as for Letters. Those articles principally reviewing a topic, those that are just as statement of opinion, and those primarily discussing the author's own work will not be considered. Articles that are successful usually present a quantitative analysis, as a way of introducing a new perspective in ecology. Authors interested in submitting such a manuscript should first send a one-paragraph proposal (no more than 300 words) to the Editorial Office (see above). 

> **Reviews and Syntheses**. Ecology Letters seeks Reviews and Syntheses on fast moving and important topics that merit rapid consideration and publication. Reviews & Syntheses are freely available online. Reviews and Syntheses can include the traditional literature review, along with more quantitative meta-analyses, syntheses, and modelling approaches, so long as those are placed in a broad context appropriate for the topic being covered. Reviews and Syntheses are expected to be of broad interest to the readership of Ecology Letters, should focus on novel principles emerging over the past several years, and should include critical evaluation, synthesis, and/or prospects for future research directions. Authors interested in submitting such a manuscript should first send a proposal to the Editorial Office (see above).

**Comment by gottacatchenall** [comment]
still working on cutting words atm, but i think 7500 shouldn't be too hard

**Comment by gottacatchenall** [comment]
@TanyaS08 so the combined figure 1? what do you think

**Comment by TanyaS08** [comment]
I think yeah lets go with combined and see what the feedback looks like?

**Comment by gottacatchenall** [comment]
@TanyaS08  do you want to close this out and start a new issue tracker for authors/AG/DG comments? also i have a sense that the primer section could feasibly moved before the roadmap, but seems like too big of a change for now

**Comment by TanyaS08** [comment]
AS in keep a 'running score' of future issues? Yeah deffos. Also I'm making grammar comments on the .html should I just do in the .md. I'll then just flag phrasing things?

**Comment by TanyaS08** [comment]
I think the primer flip could fly but lets leave as is

**Comment by gottacatchenall** [comment]
yeah sounds good

**Comment by TanyaS08** [comment]
Okay closing this issue and we can open a new one for AG/DG feedback (this is so close to AC/DC it hurts)
